### ISSUE TITLE
A11y_WINFORMS_PrintingTesting_Settings_Zoom : At 200 percent zoom the Content at the bottom of the tab like Ok and Cancel button is not visible.

### ISSUE THREAD

**Issue by adbhosale** [dotnet/winforms#7351]
### .NET version

.NET Framework Version 4.8.04161
Operating System: Windows 11 Enterprise 21H2

### Did it work in .NET Framework?

No

### Issue description

GitHubTags:#A11yMAS;#A11yTCS;#.NET Core[WINFORMS]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.4;#DesktopApp;#Win11;#Benchmark;#Zoom

Actual:
At 200 percent zoom the Content at the bottom of the tab like Ok and Cancel button is not visible.

**This issue is observed for every dialog box. The bottom content is not visible.**

Expected:
At 200 percent zoom the Content at the bottom of the tab like Ok and Cancel button should be visible.

User Impact:
User with low vision will get difficulty to see the content.

### Steps to reproduce

1. Open Core testing solution in VS 2019 Int Preview
2. Right click on cs.proj file and click on Build.
3.Then run the application. 
4.Main Form screen would appear.
5.Click on Printing_Controls:Testing the controls under Printing Tab.
6.Navigate to Setting button.
7.Setting with controls Screen would appear.
8.Use custom scaling at 200 percent and check if those buttons are visible or not.

![A11y_WINFORMS_PrintingTesting_Settings_Zoom](https://user-images.githubusercontent.com/93637360/175870437-e95ce554-2cb5-43d3-8a48-9ffe8b60e000.png)

**Comment by merriemcgaw** [comment]
@Olina-Zhang  can you get us full size screen shots of each of these print related screens, so that we can determine which we own and can fix.

**Comment by Olina-Zhang** [comment]
@merriemcgaw I verified these print related screens on display resolution: 1366*768 used in another issue, set Magnifier with 200% custom scaling. In this mode, although screens are not fully visible, and can move mouse to see all screens easily, please see following gif:
![200ZoomPrint](https://user-images.githubusercontent.com/26474449/176408356-7823c042-89ef-40b1-bb9b-b03f6dc0cae9.gif)

**Comment by merriemcgaw** [comment]
@Olina-Zhang can you show me what it looks like at 200% scaling with the resolution at 1024x768? I need to see the bottoms of the dialogs to see if they're visible in that configuration. Thanks so much!

**Comment by Olina-Zhang** [comment]
@merriemcgaw Here is the GIf result when Windows is 200% scaling with the resolution at 1024x768, have similar result with 1366*768. Can move mouse to see all screens easily.
![200ZoomPrints](https://user-images.githubusercontent.com/26474449/176580704-7741cb2e-c9ea-4b9c-979d-ac315cb1c28e.gif)

**Comment by merriemcgaw** [comment]
All of our dialogs are OK if I set the window size to 1280x1024 and change the display scaling (DPI scaling) to 200%. This should be meeting the criteria. @adbhosale can you please retest all of your scaling samples at 1280x1024? That is an acceptable value and all of our dialogs look just fine then.

**Comment by adbhosale** [comment]
Verified the bug as per above comment issue is not reproducing. Hence closing the bug
![Fixed](https://user-images.githubusercontent.com/93637360/179249834-4198112c-ba0f-4630-80a5-3836da754438.png)

**Comment by Tanya-Solyanik** [comment]
Closing per request from Samhitha K S (TATA CONSULTANCY SERVICES LTD) 

Issue is fixed. Adding #Closed; tag.
